File: examples/petstore/oseg_generated/php/UploadFile_default_example.php
<?php

require_once __DIR__ . '/vendor/autoload.php';

$config = OpenAPI\Client\Configuration::getDefaultConfiguration();

try {
    $response = (new OpenAPI\Client\Api\PetApi($config))->uploadFile(
        pet_id: 12345,
        additional_metadata: "Additional data to pass to server",
        file: new SplFileObject("/path/to/file"),
    );

    print_r($response);
} catch (OpenAPI\Client\ApiException $e) {
    echo 'Exception when calling Pet#uploadFile: '
        . print_r($e->getResponseObject());
}

File: examples/petstore/oseg_generated/csharp/UploadFileDefaultExample.cs
using System;
using System.Collections.Generic;
using System.IO;

using Org.OpenAPITools.Api;
using Org.OpenAPITools.Client;
using Org.OpenAPITools.Model;

public class UploadFileDefaultExample
{
    public static void Main()
    {
        var config = new Configuration();

        try
        {
            var response = new PetApi(config).UploadFile(
                petId: 12345,
                additionalMetadata: "Additional data to pass to server",
                file: new FileStream(
                    path: "/path/to/file",
                    mode: FileMode.Open
                )
            );

            Console.WriteLine(response);
        }
        catch (ApiException e)
        {
            Console.WriteLine("Exception when calling Pet#UploadFile: " + e.Message);
            Console.WriteLine("Status Code: " + e.ErrorCode);
            Console.WriteLine(e.StackTrace);
        }
    }
}
